plugins/mcreator-localization/help/fr_FR/potion/bad.md

```diff
@@ -1,3 +1,3 @@
-Check this parameter in case if your effect is bad for the player. 
+Vérifiez ce paramètre au cas où votre effet serait mauvais pour le joueur.

```

Comment:
In this case, "check" means "cochez", not "vérifiez".
Correction: "Cochez ce paramètre si votre effet est mauvais pour le joueur.